Makefile

```diff
@@ -120,10 +120,7 @@ test-e2e::
 	@$(BIN)/jest test-e2e
 
 test-e2e-slow::
-	@echo "Running test suite: e2e (slow tests)"
-	@node ./test-e2e-slow/release.test.js
-	@node ./test-e2e-slow/build-top-100-opam.test.js
-	@node ./test-e2e-slow/install-npm.test.js
+	@node ./test-e2e-slow/run-slow-tests
```

Comment:
@andreypopp - are you OK refactoring this to use a common `run-slow-tests` node script?

Reply:
Yep, thank you!